src/ansys/simai/core/api/design_of_experiments.py

```diff
@@ -35,13 +36,15 @@ class DesignOfExperimentsMixin(ApiClientMixin):
     def download_design_of_experiments(
         self,
         file: Optional[File],
+        dir: Optional[str | Path],
```

Comment:
Well, seeme like the CI workflows we got from the pyansys team doesn't actually run the tests, 'cause that's not valid python 3.9.
https://github.com/ansys/pysimai/actions/runs/8941539158/job/24562116546?pr=50